examples/deeplinking.py

```diff
@@ -71,6 +81,23 @@ def deep_linked_level_2(update: Update, context: CallbackContext) -> None:
 
 def deep_linked_level_3(update: Update, context: CallbackContext) -> None:
     """Reached through the USING_ENTITIES payload"""
+    update.message.reply_text(
+        "It is also possible to make deep-linking using InlineKeyboardButtons.",
+        reply_markup=InlineKeyboardMarkup(
+            [[InlineKeyboardButton(text="Like this!", callback_data=KEYBOARD_CALLBACKDATA)]]
+        ),
+    )
+
+
+def deep_link_level_3_callback(update: Update, context: CallbackContext) -> None:
+    """Answers CallbackQuery with deeplinking url."""
+    bot = context.bot
+    url = helpers.create_deep_linked_url(bot.get_me().username, USING_KEYBOARD)
```

Comment:
doesn't `Bot` have a convenience property for the username? 🤔

Reply:
You're right, i just followed how it was done for other deeplinking levels without thinking, will change 👍